src/components/Circular Support Indicator/CircularSupportIndicator.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
I would suggest to keep consistency and call this folder CircularSupportIndicator. It's really small thing, but just to keep it nice and tidy